sources/react/template-individual-icon.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+
+const needsOffset = ( name, icons ) => icons.indexOf( 'gridicons-' + name ) >= 0;
+
+const prepareIndividualIcon = ( {
+	name,
+	component,
+	iconsThatNeedOffset,
+	iconsThatNeedOffsetX,
+	iconsThatNeedOffsetY,
+} ) => `
+/* @ssr-ready */
```

Comment:
The `/** @ssr-ready **/` pragma is obsolete -- let's remove it while we're here (and add a note to the `Changelog` 😄)

Reply:
Done at d5963b9